app/test/unit/healthcheck/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var
+  test = require('tape'),
+  supertest = require('supertest'),
+  app = require('healthcheck'),
+  arrayIntersect = require('array-intersection'),
+  fs = require('fs');
+
+var 
+  pkg = require(__dirname + '/../../../../package.json'),
```

Comment:
Is there a good reason not to use `rootrequire` here? https://github.com/ericelliott/isomorphic-express-boilerplate#requiring-modules